projects.html

```diff
@@ -25,8 +25,9 @@ <h2>Completed</h2>
         {% assign advisory = site.projects | where: "label", "advisoryBoardProposals" | first %}
         <div class="col-md-12">
             <div class="advisory block" style="font-size: 20px; line-height: 30px; margin: 40px 0 80px 0;">
-                <h2>{{ advisory.name }}</h2>
-                {{ advisory }}
+                <h2>Accepted Advisory Board Proposals</h2>
+                See <a href="https://github.com/scalacenter/advisoryboard/tree/master/proposals">here</a> the list
+                of Advisory Board proposals and their respective status.
```

Comment:
This list shows all proposals, regardless of whether they've been accepted or not, whereas the old version only displayed accepted proposals. Is this what we want? Is the list of rejected proposals of broad interest? IMO pointing to the list of accepted proposals (https://github.com/scalacenter/advisoryboard/tree/master/recommendations) would be more appropriate. @darjutak WDYT?

Reply:
but note that the proposals directory has a README.md file with all the proposal statuses (I just submitted https://github.com/scalacenter/advisoryboard/pull/60 to bring it up to date), but the recommendations directory currently doesn't (though I suppose it could)

Reply:
Unfortunately, https://github.com/scalacenter/advisoryboard/tree/master/recommendations does not allow readers to browse the proposal contents because currently the GitHub interface does not follow symlinks.

I’ve updated the page to say “Advisory Board Proposals” instead of “Accepted Advisory Board Proposals”. As Seth mentioned, the full proposal list has a nice README file that shows their status, and the proposals can be read by clicking on them.

Reply:
@sjrd What do you think?

Reply:
I agree.